src/main/angular/model/error_message.ts

```diff
@@ -0,0 +1,5 @@
+/** Represents an error sent by the server */
+export class ErrorMessage {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(public message: string, public exception: any) {}
```

Comment:
since you'll be throwing these, consider making a model for the Exception.

Reply:
I'm not entirely sure what you mean by this. Do you mean making a general `Exception` class and adding subclasses?